source/main/Application.cpp

```diff
@@ -365,10 +365,10 @@ const char* CONF_VEGET_50PERC       = "50%";
 const char* CONF_VEGET_FULL         = "Full (best looking, slower)";
 
 const char* CONF_GEARBOX_AUTO       = "Automatic shift";
-const char* CONF_GEARBOX_SEMIAUTO   = "Manual shift - Auto clutch";
-const char* CONF_GEARBOX_MANUAL     = "Fully Manual: sequential shift";
+const char* CONF_GEARBOX_SEMIAUTO   = "Manual shift: auto clutch";

```

Comment:
ouch, these can't be changed - they are saved in people's config files. The reason for not using simple mnemonics is historical.